common/src/main/java/com/wynntils/features/ui/WynncraftButtonFeature.java

```diff
@@ -66,6 +71,22 @@ private void addWynncraftButton(TitleScreen titleScreen) {
         titleScreen.addRenderableWidget(wynncraftButton);
     }
 
+    /* connects to the server on "startup" - off by default */
+    private void autoConnect() {
+        if (!initialBoot && autoConnect.get()) {
```

Comment:
Can this config check be moved to where autoConnect() is called, so it forms a natural flow? Also, we don't need to add the wynncraft button if we are immediately joining the server, so maybe put that in some check as well.

Reply:
Moving the check is enough, no need to complicate it further